docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+name: ultrasinger
+services:
+    ultrasinger:
+        deploy:
+            resources:
+                reservations:
+                    devices:
+                        - driver: nvidia
+                          count: all
+                          capabilities:
+                              - gpu
+        stdin_open: true
+        tty: true
+        container_name: UltraSinger 
+        volumes:
+            - ./src/output:/app/src/output
```

Comment:
I dont like that the user has to add src folder. 
Maybe ./ultrasinger/output? or just ./output ?

Reply:
i have adjested it just testing it now

Reply:
it is now working with /output
you do need to add 
`-o /output` the the command now

Reply:
why not `- ./output:/app/src/output` ? So we doesent need the `-o` command.

Reply:
this is a oversite with me. i was having trouble getting the mount point right. i forgot to undo it